advanced_samples/CSV/dotnet/CSVtoOCS_pkce/CSVtoOCS_pkce.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <Version>1.0.0</Version>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="CsvHelper" Version="15.0.1" />
+    <PackageReference Include="IdentityModel.OidcClient" Version="3.1.2" />
+    <PackageReference Include="Microsoft.Extensions.Configuration" Version="3.1.2" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="3.1.2" />
+    <PackageReference Include="OSIsoft.OCSClients" Version="3.0.2-prerelease-20190924.3" />
```

Comment:
Should add stylecop and fxcop. If you prefer, I can go back and add those in the polaris branch later.

Reply:
If you do add those - don't bother moving all the strings to resource files, I disabled that rule in the polaris branch.